### DISCUSSION TITLE
Json Type relation POST/user.

### DISCUSSION THREAD

**Discussion by Jeffr-K** [prisma/prisma#11377] [Q&A] [answered]
Hello, All. I am Prisma new User. I am confusing about relation CRUD.

```typescript

model User
id      Int     @default(autoincrement())
name String @default("").db.VarChar(50)
Post  Post[]

model Post
id      Int     @default(autoincrement())
title   String
content String
```

and then
``` typescript
// user-service

const { name, Post } = req.body
const user = prisma.user.create({
  data: {
    name.
    Post: {
       create: {
          title: Post.??  // 
          content: Post.??
       }
    }
  }
});
```

I will send to own express server, like this:
```json
"name": "jeff",
"post": {
     "title": "Prisma Thank you all."
     "content": "Prisma is a awesome project for my life."
  }
```

Sorry, I am newbie for this. Thank you all developer.

**Comment by Jeffr-K** [comment, marked as answer]
In summary, I was not read all of prisma document for this. 

Here is link -> https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest

Happy coding.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
